fairmq/shmem/UnmanagedRegion.h

```diff
@@ -186,6 +187,19 @@ struct UnmanagedRegion
 
     bool RemoveOnDestruction() { return fRemoveOnDestruction; }
 
+    RefCount* MakeRefCount()
+    {
+        RefCount* refCount = fRefCountPool->allocate(1).get();
+        new (refCount) RefCount(1);
+        return refCount;
+    }
```

Comment:
```suggestion
    RefCount& MakeRefCount()
    {
        RefCount* refCount = fRefCountPool->allocate(1).get();
        new (refCount) RefCount(1);
        return *refCount;
    }
```

If we always return a valid `RefCount`, then let's not use raw pointer, which will technically force a caller to always check for `nullptr`.

Reply:
Done